xml/System.Data.Objects/ObjectQuery.xml

```diff
@@ -517,7 +517,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Examples  
- The example in this topic is based on the [AdventureWorks Sales Model](https://msdn.microsoft.com/library/f16cd988-673f-4376-b034-129ca93c7832).  
+ The example in this topic is based on the [AdventureWorks Sales Model](https://archive.codeplex.com/?p=msftdbprodsamples).  
```

Comment:
That page does contain a download archive link, but also points you to https://github.com/Microsoft/sql-server-samples/releases/tag/adventureworks.

Reply:
To avoid some merge conflicts, I'll change this after I merge all the related PRs.